archive/imo/imo2008_q4.lean

```diff
@@ -0,0 +1,135 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import data.real.basic
+import data.real.sqrt
+
+open real
```

Comment:
The `open real` has to be moved after the module docstring for the linter to recognise it.

Reply:
done!